lib/action_subscriber/babou.rb

```diff
@@ -0,0 +1,103 @@
+module ActionSubscriber
+  module Babou
+    ##
+    # Class Methods
+    #
+    def self.configure_from_yaml_and_cli(cli_options = {})
+      env = ENV["RAILS_ENV"] || ENV["RACK_ENV"] || ENV["APP_ENV"] || "development"
+
+      yaml_config = {}
+      babou_absolute_config_path = ::File.expand_path(::File.join("config", "babou.yml"))
```

Comment:
@mmmries Why not just rename this to `action_subscriber.yml` right now? I think it would be easy to change the config when we switch the CLI away from babou. Doesn't really make sense to support code that was never public.